nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/admin/admin-ai-export.html

```diff
@@ -0,0 +1,169 @@
+                </nuxeo-data-table-column>
+                <nuxeo-data-table-column name="[[i18n('admin.ai.stats.type')]]" flex="10">
+                    <template>
+                        [[item.type]]
+                    </template>
+                </nuxeo-data-table-column>
+                <nuxeo-data-table-column name="[[i18n('admin.ai.stats.value')]]" flex="50">
+                    <template>
+                        [[_displayVal(item)]]
+                    </template>
+                </nuxeo-data-table-column>
+            </nuxeo-data-table>
+        </nuxeo-card>
+        <nuxeo-card heading="[[i18n('admin.ai.export')]]">
+            <div class="row-container">
+               [[export]]
+            </div>
+        </nuxeo-card>
+    </template>
+    <script>
+        Polymer({
+            is: 'nuxeo-admin-ai-export',
+            behaviors: [Nuxeo.LayoutBehavior],
+            properties: {
+                in_props: {
+                    type: String,
+                    value: "dc:title,file:content"
+                },
+                out_props: {
+                    type: String,
+                    value: "dc:lastContributor"
+                },
+                split_prop: {
+                    type: String,
+                    value: "75"
+                },
+                query: {
+                    type: String,
+                    value: "SELECT * FROM Document WHERE ecm:primaryType = 'File'"
+                },
+                export: {
+                    type: String,
+                    value: ""
+                },
+                stats: {
+                    type: String
+                },
+                _total: {
+                    type: Object,
+                    computed: '_computeTotal(stats.splices)'
+                },
+                _count: {
+                    type: Object,
+                    computed: '_computeCount(stats.splices)'
+                }
+            },
+
+            _displayVal: function(val) {
+                var toReturn = "";
+                if (val.numericValue != null) {
+                    return val.numericValue;
+                }
+                if (val.type == "terms") {
+                    for (i = 0, len = val.value.length; i < len; i++) {
+                        toReturn += " " + val.value[i].key + " (" + val.value[i].docCount + ")";
+                    }
+                } else {
+                    return JSON.stringify(val.value);
+                }
+                return toReturn;
+            },
+            _filter: function(items) {
+                if (items) {
+                    return items.filter(function (item) {
+                        return !(item.type === 'total' || item.type === 'count');
+                    });
+                }
+            },
+            _computeCount: function() {
+                if (this.stats) {
+                    return this.stats.find(function (item) {
```

Comment:
nit: instead of the `if`:
```suggestion
                    return this.stats && this.stats.find(function (item) {
```